src/main/java/net/dries007/tfc/objects/items/ItemsTFC.java

```diff
@@ -114,6 +115,9 @@
     @GameRegistry.ObjectHolder("powder/salt")
     public static final ItemPowder SALT = getNull();
 
+    @GameRegistry.ObjectHolder("wooden_bucket")
```

Comment:
Object holder annotations not necessary if the field name matches the registry name (case insensitive)